### DISCUSSION TITLE
Where should boosts be?

### DISCUSSION THREAD

**Discussion by cursorweb** [cursorweb/Cycle-Bot-Game#32] [Ideas]
There are currently two ways to obtain boosts:
- Through the shop
- Through items
Here are the Pros and cons of both

# Shop
|Pros|Cons|
|--|--|
|Easy to access|Not really scalable|
|Items are organized|May need to refactor my code|

# Items
|Pros|Cons|
|--|--|
|Easy to maintain|Not organized|
|Easy to interface with|Removes the idea of 'boosts' in general|

**Comment by cursorweb** [comment]
@Codemonkey51 @FunnyLamma @alanlichen what do you think?

**Reply by alaninnovates** [reply]
I think you should be able to buy boosts through a new shop (just like the idle, cpp, text shops)

**Comment by darkdarcool** [comment]
I think maybe through random change after coding? And maybe do some sort of special challenge(or actually answer a fake repltalk question? I think that would be cool)

**Reply by cursorweb** [reply]
so you get them through coding? Like cookie clicker golden cookies? Seems like a good idea!

And for the challenges, yeah there should be like an ask command where you can do stuff for cycles

**Reply by darkdarcool** [reply]
yea, like maybe you can make so fake questions on some difficulty and see if they get it right? that would be cool.(and make different rarities of cookies)

**Reply by darkdarcool** [reply]
and if you get the ask question wrong there should be a punishment, like low self esteem or you start to second guess yourself as a coder

**Reply by darkdarcool** [reply]
and @cursorweb, can we continue this conversation in the discord so we don't chat here?

**Reply by cursorweb** [reply]
sure
